src/workerd/io/actor-cache.h

```diff
@@ -319,6 +324,9 @@ class ActorCache final: public ActorCacheInterface {
     // The value was set by the app via put() or delete(), and we have not yet initiated a write
     // to disk. The entry is appended to `dirtyList` whenever entering this state.
     //
+    // Alternatively, the app requested a get, and while we aren't "writing" anything to storage,
```

Comment:
Hmm, this is a pretty gross overloading of what it means to be `DIRTY` -- there's nothing here that needs to be written to storage, which is what dirty traditionally means in similar systems.

I'll get a better idea of how important this is after reading the rest of the code, but this really feels like it merits a new Status, or reusing one of the others like `CLEAN` or `NOT_IN_CACHE`.